Docker/arch/arch-lts-vars.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_comment": "Build with `packer build -var-file=arch-lts-vars.json arch.json`",
+  "docker_image": "saltstack/ci-arch:base",
+  "docker_repository": "saltstack",
+  "docker_image_prefix": "ci",
+  "build_type": "ci",
+  "os_name": "Arch",
+  "os_version": "lts",
+  "bootstrap_args": "-qXU -x python2",
+  "bootstrap_version": "2018.3.4",
+  "salt_branch": "develop",
```

Comment:
These should all change to master right?

Reply:
right :/

Reply:
Fixed